# License identified in include/uapi/linux/tcp.h:
/* SPDX-License-Identifier: GPL-2.0+ WITH Linux-syscall-note */

--- a/include/uapi/linux/tcp.h.orig	2023-11-03 20:28:45.879043073 -0700
+++ b/include/uapi/linux/tcp.h	2023-11-03 20:31:20.483038107 -0700
@@ -176,6 +176,7 @@ enum tcp_fastopen_client_fail {
 #define TCPI_OPT_ECN_SEEN	16 /* we received at least one packet with ECT */
 #define TCPI_OPT_SYN_DATA	32 /* SYN-ACK acked data in SYN sent or rcvd */
 #define TCPI_OPT_USEC_TS	64 /* usec timestamps */
+#define TCPI_OPT_ECN_LOW	128 /* Low-latency ECN configured at init */
 
 /*
  * Sender's congestion state indicating normal or abnormal situations
